src/store/fs.rs

```diff
@@ -84,15 +83,16 @@ use bao_tree::{
 };
 use bytes::Bytes;
 use delete_set::{BaoFilePart, ProtectHandle};
+use entity_manager::{EntityManagerState, SpawnArg};
```

Comment:
So this is the only usage of the `entity_manager` for now?

Reply:
Yeah, I wrote it generically just so I don't have to mix up all the recycling logic with the actual user code in blobs. I mean, I think it would also be an useful lib, but we can take it out and publish it later.